artemis-commons/src/test/java/org/apache/activemq/artemis/utils/ThreadLeakCheckRule.java

```diff
@@ -257,11 +262,13 @@ private boolean isExpectedThread(Thread thread) {
    @Override
    public void testSucceeded(Description description) {
       this.enabled = true;
```

Comment:
remove this.enabled from here.

There's a method to disable the leak.. you would be breaking it here.

Reply:
Hmmmm.. you removed disable(). why?